scripts/package.ps1

```diff
@@ -1,5 +1,11 @@
+$OS = Get-WmiObject -Class win32_OperatingSystem -namespace "root\CIMV2"
+
 Enable-RemoteDesktop
-Set-NetFirewallRule -Name RemoteDesktop-UserMode-In-TCP -Enabled True
+if ($OS.Version -eq "6.1.7601") {
```

Comment:
Rather than testing the explicit version. Could we test the existence of the rule?